package.json

```diff
@@ -20,10 +19,14 @@
     "worker-loader": "^2.0.0"
   },
   "scripts": {
-    "start": "batfish start",
-    "build": "batfish build",
-    "serve-static": "batfish serve-static"
+    "start": "underreact start",
+    "build": "underreact build",
+    "serve-static": "underreact serve-static"
   },
   "author": "Mapbox",
-  "license": "ISC"
+  "license": "ISC",
+  "devDependencies": {
+    "@mapbox/babel-preset-mapbox": "^7.0.1",
```

Comment:
I don't think there is a need to have an external `babel.config.js`. Read https://github.com/mapbox/underreact#babel

Reply:
Oh I totally misunderstood that section. Thanks for pointing out!

Reply:
Let us also remove the `package.json` entry for ` "@mapbox/babel-preset-mapbox"`.